src/FakeItEasy/OutputWriterExtensions.cs

```diff
@@ -50,7 +50,7 @@ public static IOutputWriter Write(this IOutputWriter writer, object value)
             Guard.AgainstNull(writer, nameof(writer));
             Guard.AgainstNull(value, nameof(value));
 
-            writer.Write(value.ToString());
+            writer.Write(value.ToString() ?? string.Empty);

```

Comment:
ToString() could theoretically return null

Reply:
seems rude of it